src/interfaces/global.ts

```diff
@@ -40,6 +41,12 @@ export interface Client {
     key?: string;
 }
 
+export interface Board {
```

Comment:
way you don't add all the different properties of board ?
(even that not all the boards have this properties you can add "?" for optional properties e.g. token?:string)

Reply:
Don't think somebody will ever use them. It's kind of garbage

Reply:
You are right about token - missed it. 
Added "token" only. The rest is a garbage - no one will ever use them.